src/renderer/components/ft-prompt/ft-prompt.js

```diff
@@ -64,6 +67,11 @@ export default defineComponent({
     })
     this.focusItem(0)
   },
+  destroyed() {
+    if (this.lastActiveElement != null) {
+      this.lastActiveElement.focus()
+    }
```

Comment:
```suggestion
   this.lastActiveElement?.focus()
```

Reply:
Lots of examples of this, so just do a find and replace

Reply:
I only see 3 (search by `Element.focus()`)